cmd/server/handlers.go

```diff
@@ -271,3 +271,46 @@ func GetTeamsHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write(json)
 	log.Printf("%s %s %v", r.Method, r.URL, time.Since(start))
 }
+
+func MessagePreviewHandler(w http.ResponseWriter, r *http.Request) {
+	EnableCors(&w)
+	start := time.Now()
+
+	id := mux.Vars(r)["id"]
+	goal, _ := dao.GetGoal(id)
+
+	// TODO: This is nearly duplicated with the code in getGoals and getGoal
+	videoUrl, err := s3Client.NewSignedGetURL(goal.S3ObjectKey, config.AwsBucketName, time.Hour*24*7)
+	if err != nil {
+		log.Println(err)
+	}
+	thumbnailUrl, err := s3Client.NewSignedGetURL(goal.ThumbnailS3Key, config.AwsBucketName, time.Hour*24*7)
```

Comment:
`time.Hour*24*7`
Would be nice to extract this to a constant

Reply:
Yeah that will probably go along with the TODO to extract this presigned url stuff to a single location.